cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v2/transactions.rst

```diff
@@ -0,0 +1,38 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _http-restful-api-transactions:
+
+====================================
+Transaction Service HTTP RESTful API
+====================================
+
+Some internals of the Transaction Service are exposed via HTTP REST endpoints.
```

Comment:
Certain internals of the CDAP Transaction Service are exposed via HTTP RESTful endpoints and are documented here.